packages/core/src/components/ColorPicker/ColorPicker.styles.tsx

```diff
@@ -28,29 +29,15 @@ export const styles: Partial<
     display: "flex",
     justifyContent: "center",
     padding: theme.colorPicker.panelPadding,
-    border: theme.colorPicker.panelBorder,
-    borderRadius: theme.colorPicker.panelBorderRadius,
```

Comment:
I think these two props can now be remove from the theme as they're no longer used:
https://github.com/lumada-design/hv-uikit-react/blob/master/packages/styles/src/types.ts#L359
https://github.com/lumada-design/hv-uikit-react/blob/master/packages/styles/src/theme.ts#L352
https://github.com/lumada-design/hv-uikit-react/blob/master/packages/styles/src/themes/ds3.ts#L568
https://github.com/lumada-design/hv-uikit-react/blob/master/packages/styles/src/themes/ds5.ts#L511

Reply:
that's right 👍